lisp/Makefile

```diff
@@ -37,9 +43,44 @@ all: julia-mode.el $(ELC) ess-custom.el ess-autoloads.el
 .PHONY: dist
 dist: all
 
-install: dist
+dist/package.out: 
+	$(INSTALLDIR) dist
+	if [ ! -s "dist/$(PKBUILD).tar.gz" ] ; then \
+	  cd dist ; curl -sLOk https://github.com/melpa/package-build/archive/$(PKBUILD).tar.gz ; fi
+	cd dist ; tar xfz $(PKBUILD).tar.gz
+	cd dist/package-build-$(PKBUILD) ; make -s loaddefs
+	$(INSTALLDIR) dist/recipes
+	if [ ! -s "dist/recipes/ess" ] ; then \
+	  cd dist/recipes ; curl -sLOk https://raw.githubusercontent.com/melpa/melpa/master/recipes/ess ; fi
+	$(EMACSBATCH) -L dist/package-build-$(PKBUILD) \
+	--eval "(require 'package-build)" \
```

Comment:
Let's extract this spaghetti into a separate .el file inside etc/targets/ subfolder.

Reply:
done... for lack of better place moved to `lisp/tmp/ess-package.el`